src/_gettsim/transfers/arbeitsl_geld_2/kost_unterk.py

```diff
@@ -132,19 +132,79 @@ def _arbeitsl_geld_2_berechtigte_wohnfläche_bg(
     params = arbeitsl_geld_2_params["berechtigte_wohnfläche_eigentum"]
     max_anzahl_direkt = params["max_anzahl_direkt"]
     if bewohnt_eigentum_hh:
-        if anz_personen_hh <= max_anzahl_direkt:
-            maximum = params[anz_personen_hh]
+        if anz_personen_bg <= max_anzahl_direkt:
+            maximum = params[anz_personen_bg]
         else:
             maximum = (
                 params[max_anzahl_direkt]
-                + (anz_personen_hh - max_anzahl_direkt) * params["je_weitere_person"]
+                + (anz_personen_bg - max_anzahl_direkt) * params["je_weitere_person"]
             )
     else:
         maximum = (
             arbeitsl_geld_2_params["berechtigte_wohnfläche_miete"]["single"]
-            + max(anz_personen_hh - 1, 0)
+            + max(anz_personen_bg - 1, 0)
             * arbeitsl_geld_2_params["berechtigte_wohnfläche_miete"][
                 "je_weitere_person"
             ]
         )
-    return min(wohnfläche_hh, maximum)
+    return min(wohnfläche_bg, maximum)
+
+
+def bruttokaltmiete_m_bg(
+    bruttokaltmiete_m_hh: float,
+    _anteil_personen_in_haushalt_bg: float,
+) -> float:
+    """Share of household's monthly rent attributed to the Bedarfsgemeinschaft.
```

Comment:
Could you add a reference for this rule? Doesn't seem obvious, i.e., there could be distinctions by adults/children, whatever. Same for the other two functions.

Reply:
Added it. Excerpt BSG ruling:
Die Aufteilung der Kosten der Unterkunft und Heizung bei der Prüfung des Kinderzuschlags erfolgt nur für Mitglieder der Bedarfsgemeinschaft nach dem Existenzminimumbericht der Bundesregierung, hinsichtlich der anderen Haushaltsmitglieder jedoch nach Kopfteilen.